.github/workflows/ci.yml

```diff
@@ -17,6 +17,12 @@ jobs:
       run: swift test -c debug
     - name: Test in Release
       run: swift test -c release
+    - name: Run on Xcode 12.0
+      run: sudo xcode-select -switch /Applications/Xcode_12.app
+    - name: Test in Debug
+      run: swift test -c debug
+    - name: Test in Release
+      run: swift test -c release
```

Comment:
Should we generate the linux tests too for completeness?

Reply:
I don't think so since we already did in the step before? We generate the tests so they can be run in the linux jobs, so creating them twice seems unnecessary. We could probably delete that step too if we adopt https://github.com/airbnb/ResilientDecoding/issues/32

Reply:
Makes sense.